tools/project-context.js

```diff
@@ -89,9 +90,11 @@ _.extend(ProjectContext.prototype, {
     // writes to; used by `meteor test` so that you can test your
     // app in parallel to writing it, with an isolated database.
     // You can override the default .meteor/local by specifying
-    // METEOR_LOCAL_DIR.
+    // METEOR_LOCAL_DIR. You can use relative path if you want it
+    // relative to your project directory.
     self.projectLocalDir = process.env.METEOR_LOCAL_DIR ?
-      process.env.METEOR_LOCAL_DIR : (options.projectLocalDir ||
+      path.resolve(options.projectLocalDir, process.env.METEOR_LOCAL_DIR)
+      : (options.projectLocalDir ||
```

Comment:
for consistency with the rest of the codebase this should be `files.pathResolve(options.projectLocalDir, files.convertToStandardPath(process.env.METEOR_LOCAL_DIR))`